extensions/investspec/test/test_investspec.py

```diff
@@ -14,13 +14,13 @@
 class TestInvestSpec(unittest.TestCase):
 
     @classmethod
-    def beforeAll(cls):
+    def setUpClass(cls):
```

Comment:
This function wasn't running because I named it wrong, `beforeAll` is the equivalent for Jest. 🤯